docs/includes/openapi/openapi-generator.adoc

```diff
@@ -0,0 +1,543 @@
+
+==== Using the Online Generator
+
+The OpenAPI tools project hosts and maintains the online OpenAPI generator at http://api.openapi-generator.tech. You can use the site's API browser to explore the available generators and the settings each supports, expressed as JSON.
+
+To generate your project, you supply the options and additional properties as JSON. The online generator provides you with a file ID, and you refer to the file ID in a subsequent HTTP request to retrieve your project.
+
+[NOTE]
+The online generator stores your project on the server which you then retrieve using a separate HTTP request.
+Before you use the online generator, consider whether any of the input you provide--the OpenAPI document, package or Maven coordinates--and therefore the generated project will reveal any sensitive information.
+
+This document does not explore further the use of the online generator.
+
+// end::usage[]
+
+
+// tag::using-generated-code-intro[]
+[[using-generated-code]]
+== Using the Generated Code
+
+The Helidon generators go a long way in helping you write your client or server. Even so, there are important parts of your project only you can provide. This section describes your next steps _after_ you have run the generator.
+// end::using-generated-code-intro[]
+
+// tag::using-generated-code-server[]
+// tag::using-generated-code-server-intro[]
+
+=== Completing the Server
+Recall from earlier how the OpenAPI generator gathers operations into one or more "APIs" and generates either a class or an interface--your choice--for each API.
+You need to extend each generated API class or implement each generated API interface by writing your own classes.
+Any input parameters to the endpoints are expressed as POJO model objects or Java types, as declared in the OpenAPI document. Your server code uses each of the input parameters to accomplish whatever business purpose that endpoint is responsible for, possibly returning a result as a POJO or Java type as indicated for that operation in the OpenAPI document.
+// end::using-generated-code-server-intro[]
+
+In some cases, you might need more control over the response sent to the client. In that case, specify the additional property `returnResponse=true` when you run the Helidon server generator. The return type for the generated methods is
+ifdef::mp-flavor[]
+the Jakarta RESTful web services `Response`
+endif::mp-flavor[]
+ifdef::se-flavor[]
+the Helidon SE `ServerResponse`
+endif::se-flavor[]
+and your code has complete control--and therefore responsibility--over setting the status, writing the response entity (if any), and assigning any returned headers.
+
+Your code plus the server code from the Helidon generator--all running on Helidon {flavor-uc}--combine to implement fully the server API declared in the original OpenAPI document. Build your project to get a tailored Helidon {flavor-uc} server `.jar` file or Docker image and your server is ready to run.
```

Comment:
"fully implement"?

Reply:
Ah, yes whether to boldly split that infinitive! IIRC Lisa pointed me to the conventions or advised me separately that splitting infinitives is OK in Oracle doc and (to my ear) it does read a bit better with your change. Fixed.